packages/cli/lib/index.js

```diff
@@ -82,6 +82,17 @@ prog
 	.describe('Start a live-reload server for development')
 	.option('--src', 'Specify source directory', 'src')
 	.option('--cwd', 'A directory to use instead of $PWD', '.')
+	.option('--devServer', 'Determine if dev server should be enabled', true)
+	.option(
+		'--dest',
+		'Specify output directory if dev server is disabled',
+		'build'
+	)
+	.option(
+		'--ignore',
+		'Path relative to src to be ignored during watch if dev server is disabled',
+		''
```

Comment:
This default value means that `src/` is ignored by default. I don't think this is intended?

Reply:
```suggestion
		null
```